python/trezorlib/tests/device_tests/test_msg_eos_signtx.py

```diff
@@ -0,0 +1,1132 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2018 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+import time
+import json
+
+from collections import namedtuple
+from trezorlib import eos, messages as proto
+from trezorlib.tools import parse_path
+from trezorlib.eos import parse_transaction_json, parse_action
+from .common import TrezorTest
+
+
+@pytest.mark.eos
+class TestMsgEosSignTx(TrezorTest):
+    def test_eos_signtx_transfer_token(self):
+        self.setup_mnemonic_nopin_nopassphrase()
+        data = '''{
+            "chain_id": "cf057bbfb72640471fd910bcb67639c22df9f92470936cddc1ade0e2f2e7dc4f",
```

Comment:
please make a `CHAIN_ID` constant at top level and use it in every test case

Reply:
CHAIN_ID made a constant